src/main/java/com/softserve/academy/spaced/repetition/domain/Account.java

```diff
@@ -28,13 +28,13 @@
     private Long id;
 
     @Column(name = "password")
-    @Size(message = PASS_SIZE_MESSAGE, min = PASS_MIN_SIZE, max = PASS_MAX_SIZE, groups = Request.class)
+    @Size(min = PASSWORD_MIN_SIZE, max = PASSWORD_MAX_SIZE, message = "{message.validation.fieldSizeLimits}", groups = Request.class)
     private String password;
 
     @Column(name = "email", unique = true, nullable = false)
-    @NotNull(message = NULL_MESSAGE, groups = Request.class)
-    @Size(min = EMAIL_MIN_SIZE, max = EMAIL_MAX_SIZE, message = EMAIL_SIZE_MESSAGE, groups = Request.class)
-    @Pattern(regexp = EMAIL_PATTERN, message = EMAIL_PATTERN_MESSAGE, groups = Request.class)
+    @NotNull(message = "{message.validation.fieldNotNull}", groups = Request.class)
+    @Size(min = EMAIL_MIN_SIZE, max = EMAIL_MAX_SIZE, message = "{message.validation.fieldSizeLimits}", groups = Request.class)
+    @Pattern(regexp = EMAIL_PATTERN, message = "{message.validation.emailWrongFormat}", groups = Request.class)
     @EmailNotExist(groups = Request.class)
```

Comment:
please rename validation for email
because in this context it means checking on email not exist
but we need checking on email exist